Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -356,7 +436,7 @@ def review(self, pull_request, comment=None, approve=None):
                     content=comment,
                 ))
 
-            return pull_request
+            return pull_request if is_successful else None
```

Comment:
nit: since None is implicit, you could instead write
```suggestion
            if is_successful:
                return pull_request
```